src/gerbil/runtime/hash.ss

```diff
@@ -66,6 +66,15 @@ namespace: #f
 (bind-method! __table::t 'copy raw-table-copy)
 (bind-method! __table::t 'clear! raw-table-clear!)
 
+(bind-method! __gc-table::t 'ref gc-table-ref)
```

Comment:
With so many of these, maybe a macro?

Reply:
hrm, maybe, but this is at least quite clear.